cmd/livepeer/starter/starter.go

```diff
@@ -717,12 +720,20 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 				gpm,
 				sm,
 				timeWatcher,
-				cfg,
+				tcfg,
 			)
 			if err != nil {
 				glog.Errorf("Error setting up PM recipient: %v", err)
 				return
 			}
+			mfv, _ := new(big.Int).SetString(*cfg.MaxFaceValue, 10)
```

Comment:
Should we also check the bool that indicates success here?

Reply:
My understanding was `SetString` returns `nil` if it fails.  Is this correct?  This is same pattern used on lines 675 and 740. I can update if prefer to check bool rather than value or prefer to check both.

Reply:
From looking at the `SetString` code, it seems like the way you've done it probably is safe:

> If SetString fails, the value of z is undefined but the returned value is nil

but I think to be safe and not depend on that behavior never changing, let's also check the `success` bool isn't false

Reply:
I think that for this one, for the sake of consistency, we can keep it as it is. And later convert all calls `mfv, _ := new(big.Int).SetString(*cfg.MaxFaceValue, 10)` to what Thom suggested.